tasks/firewall.yml

```diff
@@ -0,0 +1,74 @@
+- name: ensuring that ufw is installed
+  apt:
+    state: present
+    name:
+      - ufw
+
+- name: activating ingress
+  ufw:
+    policy: deny
+    direction: incoming
```

Comment:
I just realised. We need to ALLOW incoming by default.  Remember; TURN allocates random ports in the  32768-61000 range and those need to be reachable from the outside!

We need to flip the egress and the ingress rules I think....

Reply:
Please have a look at https://github.com/wireapp/ansible-restund/pull/18/commits/91b24254c03f80bdfb7057767fcdddd5aa474ece